### PR TITLE
Upcoming events section added

### LINKED ISSUE #58: Create Upcoming Event Section
**Deliverables**
- Create an Upcoming Event Component.
- For Image, as of now, you can use any non-copyrighted image.

**Sample**
Capture4

**Additional context**
- Tech/Framework:- React Js
- External Library :- You can use React Bootstrap

### PR DESCRIPTION
**Pull request by rutuja2229**
# Description

Created an  Upcoming event section

Closes #58 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Any dependent changes have been merged and published in downstream modules

## Screenshots

image

### CODE REVIEW

**Comment by rutuja2229** [comment]
@codewithvk Please take a review at this, will the colour seems good to you?
I was unable to make it similar to that mentioned in the issue, any suggestions will be helpful for me. And apologies for the delay.

File: frontend/src/Components/Upcomingeve/Upcomingevents.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import image2 from "../../images/event2.jpeg";
+import image3 from "../../images/event3.jpg";
+
+const Upcomingevents = () => {
+    return(
+        
+            <Container>
+            <Row>
+                <Col>
+                    <Card   id = 'upcomingeve' style={{ width: '18rem' }}  >
```
Comment (codewithvk) [inline frontend/src/Components/Upcomingeve/Upcomingevents.js:17]: You should create one object for all dummy data, Fetch here and through map/ Create separate component for Cart item.

**Review by codewithvk — CHANGES_REQUESTED** [review]

File: frontend/src/Components/Upcomingeve/Upcomingevents.js (hunk 1 above)
Comment (codewithvk) [reply frontend/src/Components/Upcomingeve/Upcomingevents.js:17]: @rutuja2229 make the above changes.

**Comment by rutuja2229** [comment]
Can u please check this again?

**Comment by rutuja2229** [comment]
@codewithvk can u plz reply?

**Comment by codewithvk** [comment]
@rutuja2229 Sorry for the late reply, But unfortunately this project work was stoped according to the admin statement. So humble request don't work on this project until I will inform you. If you work with on my mentoring you can check my second project by [here](https://github.com/purnima143/Kurakoo)

**Pull request merged**